Examples/WriteApiExample.cs

```diff
@@ -0,0 +1,330 @@
+                            Console.WriteLine(
+                                "-----------------------------------------------------------------------");
+                            break;
+                    }
+                };
+
+                //
+                // Write by LineProtocol
+                //
+                writeApi.WriteRecord("influxPoint,writeType=lineProtocol value=11.11" +
+                                     $" {DateTime.UtcNow.Subtract(EpochStart).Ticks * 100}", WritePrecision.Ns,
+                    "my-bucket", "my-org");
+
+                //
+                // Write by LineProtocol - bad timestamp
+                //
+                writeApi.WriteRecord("influxPoint,writeType=lineProtocol value=11.11" +
+                                     $" {DateTime.UtcNow}", WritePrecision.Ns, "my-bucket", "my-org");
+
+                //
+                // Write by Data Point
+                //               
+                var point = PointData.Measurement("influxPoint")
+                    .Tag("writeType", "dataPoint")
+                    .Field("value", 22.22)
+                    .Timestamp(DateTime.UtcNow.AddSeconds(-10), WritePrecision.Ns);
+
+                writeApi.WritePoint(point, "my-bucket", "my-org");
+
+                //
+                // Write by Data Point - bad "value" type
+                //               
+                var wrongPoint = PointData.Measurement("influxPoint")
+                    .Tag("writeType", "dataPoint")
+                    .Field("value", "22.22")
+                    .Timestamp(DateTime.UtcNow.AddSeconds(-10), WritePrecision.Ns);
+
+                writeApi.WritePoint(wrongPoint, "my-bucket", "my-org");
+
+                //
+                // Write by POCO
+                //
+                var influxPoint = new InfluxPoint { WriteType = "POCO", Value = 33.33, Time = DateTime.UtcNow };
+
+                writeApi.WriteMeasurement(influxPoint, WritePrecision.Ns, "my-bucket", "my-org");
+
+                //
+                // Write by POCO - bucket not found
+                //
+                writeApi.WriteMeasurement(influxPoint, WritePrecision.Ns, "my-bucket2", "my-org");
+
+                writeApi.Dispose();
+                while (!writeApi.Disposed) Thread.Sleep(10);
+
+                var tables = influxDbClient.GetQueryApi()
+                    .QueryAsync("from(bucket:\"my-bucket\") |> range(start: 0)", "my-org").Result;
+
+                tables.ForEach(table =>
+                {
+                    var fluxRecords = table.Records;
+                    fluxRecords.ForEach(record => { Console.WriteLine($"{record.GetTime()}: {record.GetValue()}"); });
+                });
+
+                influxDbClient.Dispose();
+            }
+
+            return Task.CompletedTask;
+        }
+
+        private static Task CustomEventListener()
+        {
+            var influxDbClient = InfluxDBClientFactory.Create("http://localhost:9999/",
```

Comment:
Please rename to to `client` and use the `using` keyword:

```suggestion
            using var client = InfluxDBClientFactory.Create("http://localhost:9999/",
```